dll/win32/netcfgx/tcpipconf_notify.c

```diff
@@ -2719,6 +2719,8 @@ LoadDNSSettings(
     {
         dwSize = sizeof(DWORD);
         RegQueryValueExW(hKey, L"RegisterAdapterName", NULL, NULL, (LPBYTE)&This->pCurrentConfig->pDNS->RegisterAdapterName, &dwSize);
+
+        dwSize = sizeof(This->pCurrentConfig->pDNS->RegistrationEnabled);
```

Comment:
Stay consistent. Everywhere else uses `sizeof(DWORD)`

Reply:
This function uses `sizeof(DWORD)` everywhere, except the case where the buffer size is nontrivial.
Your change made this simple DWORD case look like the complicated string case below, instead of looking like all the other cases that are actually like it.

This is what style is useful for: making similar things look similar, so that differences stand out.